lib/opencensus/metrics.ex

```diff
@@ -0,0 +1,88 @@
+defmodule Opencensus.Metrics do
+  @moduledoc """
+  Functions to help Elixir programmers use OpenCensus metrics.
+
+  First, a measure must be created:
+  ```elixir
+  Metrics.new_measure(:name, "description", :unit)
+  Metrics.new_measure(:another, "", :unit)
+  ```
+
+  Next step is to choose aggregations to be exported:
+  ```elixir
+  Metrics.aggregate_count(:name_count, :name, "count", [:tag1, :tag2])
+  Metrics.aggregate_gauge(:name_gauge, :name, "gauge", [:tag1, :tag2])
+  Metrics.aggregate_sum(:name_sum, :name, "sum", [:tag1, :tag2])
+  Metrics.aggregate_distribution(:name_distribution, :name, "distribution", [:tag1, :tag2], [0, 100, 1000])
+  ```
+
+  After aggregations are decided, measures may be recorded by explicitly providing tags:
+  ```elixir
+  Metrics.record(:name, %{tag1: "v1", tag2: "v2"}, 3)
+  Metrics.record([another: 1, name: 100], %{tag1: "v1", tag2: "v2})
+  ```
+  or using tag values that are present in process dictionary:
+  ```elixir
+  Metrics.record(:name, 3)
+  ```
+  """
+
+  def new_measure(name, description, unit) do
+    :oc_stat_measure.new(name, description, unit)
```

Comment:
I would prefer to use `defdelegate` instead of "handcrafted" function.

Reply:
I did not know about `defdelegate`. Thanks!